test/index.js

```diff
@@ -161,7 +162,7 @@ describe('Token', function () {
     var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', { expiresIn: -10 }) } };
 
     server.inject(request, function (res) {
-      expect(res.result.message).to.equal('Expired token received for JSON Web Token validation');
+      expect(res.result.message).to.equal('jwt expired');
```

Comment:
it is the error message from jsonwebtoken. It's not TOO descriptive but...

Do you want to wrap all errors in a more descriptive error? don't know if its worth it